k8s/models/cronjob.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+# Copyright 2017-2019 The FIAAS Authors
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import six
+
+from .common import ObjectMeta, ObjectReference, Time, ListMeta
+from .job import JobTemplateSpec
+from ..base import Model
+from ..fields import Field, ListField
+
+
+class CronJobSpec(Model):
+    concurrencyPolicy = Field(six.text_type) 
+    failedJobsHistoryLimit = Field(int)
```

Comment:
Codacy found an issue: [variable 'failedJobsHistoryLimit' in class scope should not be mixedCase (N815)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=5919885)